app/controllers/albums_controller.rb

```diff
@@ -0,0 +1,64 @@
+class AlbumsController < ApplicationController
+  def get_album
+    id = params[:id]
+    @album = Album.find(id)
+  end
+
+  def index
+    @albums = Album.all
+    @albums_sort = @albums.sort_by{ |album| album[:rank] }.reverse
+  end
+
+  def show
+    get_album
+  end
+
+  def new
+    @album = Album.new
+    @title = "Add New album"
```

Comment:
Nice job prompting the user on what to do here! To prompt the user even further, you could [use the placeholder attribute on form fields in the view](http://stackoverflow.com/questions/3259234/ruby-on-rails-how-do-i-add-placeholder-text-to-a-f-text-field) to give some example form input.